app/javascript/containers/ReviewModeContainer.jsx

```diff
@@ -108,55 +71,97 @@ class ReviewModeContainer extends React.Component {
     for (let category in this.props.categories) {
       let stateCategory = this.props.categories[category];
       count += 1;
-      categoryMap[stateCategory.id] = getQuestionsByCategory(stateCategory.id, questions);
-      stack.push(<CategoryHeader
-        category={stateCategory}
-        number={count}
-        buildingId={building.id}
-      />);
+      categoryMap[stateCategory.id] = getQuestionsByCategory(
+        stateCategory.id,
+        questions
+      );
+      stack.push(
+        <CategoryHeader
+          category={stateCategory}
+          number={count}
+          buildingId={building.id}
+        />
+      );
       stack = stack.concat(
         <table cellSpacing="0">
           <tbody>
-          {mapCategorytoQuestions(categoryMap, stateCategory.id, building)}
+            {mapCategorytoQuestions(categoryMap, stateCategory.id, building)}
           </tbody>
         </table>
       );
     }
     return stack;
   }
 
-
   render() {
+    var parentQuestionsForDelegations = this.props.questions.filter(
+      question => {
+        answer = this.props.getAnswer(question.id);
+        return answer && !answer.text && answer.delegation_email;
+      }
+    );
+
+    var delegations = [];
+    for (var i = 0; i < parentQuestionsForDelegations.length; i++) {
+      var question = parentQuestionsForDelegations[i];
+      var answer = this.props.getAnswer(question.id);
+      var allDependentQuestions = this.props.getPotentialDependentQuestions(
+        question
+      );
+      allDependentQuestions.push(question);
+
+      allDependentQuestions.map(currentQuestion => {
+        var currentAnswer = this.props.getAnswer(currentQuestion.id);
+        if (currentAnswer) {
+          var delegation = {
+            email: answer.delegation_email,
+            first_name: answer.delegation_first_name,
+            last_name: answer.delegation_last_name,
+            answer_id: currentAnswer.id
+          };
+          delegations.push(delegation);
+        }
+      });
+    }
+    let empty = delegations.length == 0 ? true : false;
```

Comment:
Oops, this is probably our fault for calling the method `submitDelegation`, but in the future we actually want the Submit Form button to take the user back to their home page and mark all of their delegated questions as completed (so they don't see them again). So even if the user clicks the button with no delegations to make, it should still succeed -- it just doesn't need to make the API call to submit new delegations. Let's keep the button enabled at all times, but add a check in `submitDelegations` to skip the request if there's nothing to do!